ccmlib/common.py

```diff
@@ -452,12 +453,7 @@ def get_stress_bin(install_dir):
         add_exec_permission(path, short_bin)
     elif not os.access(stress, os.X_OK):
         try:
-            # try to add user execute permissions
-            # os.chmod doesn't work on Windows and isn't necessary unless in cygwin...
-            if sys.platform == "cygwin":
-                add_exec_permission(path, stress)
-            else:
-                os.chmod(stress, os.stat(stress).st_mode | stat.S_IXUSR)
+            os.chmod(stress, os.stat(stress).st_mode | stat.S_IXUSR)
```

Comment:
please notice that `if sys.platform == cygwin` is branched one level earlier, so we cannot get to this place